cpp/src/aztec/proof_system/work_queue/work_queue.cpp

```diff
@@ -58,25 +58,6 @@ size_t work_queue::get_scalar_multiplication_size(const size_t work_item_number)
     return 0;
 }
 
-/**
- * Returns a boolean that denotes if the scalar multiplication is to be done with the Monomial SRS or Lagrange SRS.
```

Comment:
Removed this as the scalar multiplication type will always be monomial SRS

Reply:
If you want to maintain backwards compatibility/ some deprecation policy -- I can leave this in and have it always return false

Reply:
sounds good